pypeit/utils.py

```diff
@@ -42,6 +48,62 @@ def bspline_inner_knots(all_knots):
     i1=pos[-1]
     return all_knots[i0:i1]
 
+def fast_running_median(seq, window_size):
```

Comment:
Interesting.  I'm always interested in finding new ways to manipulating python to do things faster.  A few thoughts:
 - Probably makes sense to move this to the `pypeit.filter` module (we should remove the SDSS header).
 - May be interesting to add this as an option in `pypeit.filter.BoxcarFilter`.
 - I wonder if there's any additional speed gains in using numpy.searchsorted and numpy.insert instead of the bisect package.

Reply:
I would be fine with making this a method there. . Although I think filter.py should then probably be moved to core.

Reply:
Sounds good.  Let's push this to a later PR.